package.json

```diff
@@ -27,22 +28,24 @@
     "scripts": {
         "clean": "rimraf dist/* *.tsbuildinfo",
         "prebuild": "npm run clean && npm run prettier:check && npm run intl:compile",
-        "build:esm": "tsc --project tsconfig.esm.json",
-        "build:cjs": "tsc --project tsconfig.cjs.json",
         "build": "npm run build:esm && npm run build:cjs",
-        "copy-messages": "cp -r src/messageFiles/compiled dist/messages",
-        "build:pnpm": "npm run build && npm run intl:compile && npm run postbuild",
+        "build:esm": "tsc --project tsconfig.esm.json && babel --config-file=./babel.esm.js src/ --extensions=.ts --out-dir=dist/esm/ --out-file-extension=.mjs",
+        "build:cjs": "tsc --project tsconfig.cjs.json && babel --config-file=./babel.cjs.js src/ --extensions=.ts --out-dir=dist/cjs/",
         "watch": "tsc --watch --preserveWatchOutput --project .",
-        "postbuild": "npm run copy-messages",
         "prerelease": "npm run build",
         "lint": "eslint src",
         "prettier:check": "prettier --check './src/**/*.{ts,js}'",
         "prettier:fix": "prettier --write './src/**/*.{ts,js}'",
         "release": "np",
         "release:preview": "npm run release -- --any-branch --tag preview --no-release-draft",
-        "test": "echo \"Tests? maybe next time ;)\" && exit 0",
-        "intl:extract": "formatjs extract 'src/messageDescriptors/**/*.ts' --out-file src/messageFiles/source/en.json --format crowdin --id-interpolation-pattern '[sha512:contenthash:base64:6]'",
-        "intl:compile": "formatjs compile-folder --ast --format crowdin src/messageFiles/source src/messageFiles/compiled"
+        "test": "npm run test:build",
+        "test:build": "npm run test:build:messages && npm run test:build:cjs && npm run test:build:esm && npm run test:build:dts",
+        "test:build:cjs": "node test/test-build.cjs",
+        "test:build:esm": "node test/test-build.mjs",
+        "test:build:dts": "tsc --noEmit dist/cjs/index.d.ts && tsc --noEmit dist/esm/index.d.ts",
+        "test:build:messages": "echo 'Checking if the dist/messages directory contains 49 files' && ls dist/messages/*.json | cut -d\\/ -f3 | cut -d. -f1 | sort | wc -l | grep -q 49 || (echo 'Fail' && false) && echo 'OK'",
+        "intl:extract": "formatjs extract 'src/messages/**/*.ts' --out-file i18n/en.json --format crowdin --id-interpolation-pattern '[sha512:contenthash:base64:6]'",
+        "intl:compile": "formatjs compile-folder --ast --format crowdin i18n dist/messages"
```

Comment:
I've changed `intl:compile` script to output directly to the `dist` folder. 

This way we don't need the `copy-messages` target anymore.